packages/next-on-pages/src/buildApplication/generateGlobalJs.ts

```diff
@@ -15,8 +15,14 @@ export function generateGlobalJs(): string {
 				env: new Proxy(
 					{},
 					{
-						get: (_, property) => Reflect.get(envAsyncLocalStorage.getStore(), property),
-						set: (_, property, value) => Reflect.set(envAsyncLocalStorage.getStore(), property, value),
+						get: (_, property) => {
+							if(!envAsyncLocalStorage.getStore()) return;
```

Comment:
Why is this and L23 necessary?

Reply:
I need to check with the runtime team, I'm somewhat confident that it's a runtime bug

anyways this is why I put the PR in draft I need to dig a sec deeper into this

Reply:
I've removed this.... it doesn't seem necessary anymore... 🤷 😓

Reply:
I'm not too sure what's going on to be honest, but if this works I think we can just proceed with it this way and keep an eye out for it 🤔